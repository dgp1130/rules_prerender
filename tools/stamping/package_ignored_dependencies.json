{
  "name": "test",
  "version": "0.0.0-PLACEHOLDER",
  "dependencies": {
    "rules_prerender": "0.0.0-PLACEHOLDER",
    "ignored-dep": "0.0.0-PLACEHOLDER",
    "@ignored/other-dep": "0.0.0-PLACEHOLDER"
  },
  "devDependencies": {
    "rules_prerender": "0.0.0-PLACEHOLDER",
    "ignored-dep": "0.0.0-PLACEHOLDER",
    "@ignored/other-dep": "0.0.0-PLACEHOLDER"
  },
  "peerDependencies": {
    "rules_prerender": "0.0.0-PLACEHOLDER",
    "ignored-dep": "0.0.0-PLACEHOLDER",
    "@ignored/other-dep": "0.0.0-PLACEHOLDER"
  },
  "optionalDependencies": {
    "rules_prerender": "0.0.0-PLACEHOLDER",
    "ignored-dep": "0.0.0-PLACEHOLDER",
    "@ignored/other-dep": "0.0.0-PLACEHOLDER"
  }
}
