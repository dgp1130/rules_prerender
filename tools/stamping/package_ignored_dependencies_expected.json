{
  "name": "test",
  "version": "1.2.3",
  "dependencies": {
    "rules_prerender": "1.2.3",
    "ignored-dep": "0.0.0-PLACEHOLDER",
    "@ignored/other-dep": "0.0.0-PLACEHOLDER"
  },
  "devDependencies": {
    "rules_prerender": "1.2.3",
    "ignored-dep": "0.0.0-PLACEHOLDER",
    "@ignored/other-dep": "0.0.0-PLACEHOLDER"
  },
  "peerDependencies": {
    "rules_prerender": "1.2.3",
    "ignored-dep": "0.0.0-PLACEHOLDER",
    "@ignored/other-dep": "0.0.0-PLACEHOLDER"
  },
  "optionalDependencies": {
    "rules_prerender": "1.2.3",
    "ignored-dep": "0.0.0-PLACEHOLDER",
    "@ignored/other-dep": "0.0.0-PLACEHOLDER"
  }
}
